Lib/test/test_smtplib.py

```diff
@@ -785,7 +785,7 @@ def found_terminator(self):
             except ResponseException as e:
                 self.smtp_state = self.COMMAND
                 self.push('%s %s' % (e.smtp_code, e.smtp_error))
-                return
+            return
```

Comment:
The mis-indented return caused the test for smtpd state to "fall through" to the next test, resulting ultimately in "Internal Confusion" error.